### PR TITLE
Completed new formatting of guide in spanish

### PR DESCRIPTION
**Pull request by BlackLotus64**
Fixed the completion of the guide in chapters and not in the monero-GUI-guide MD file. Erased the changes made to `monero-GUI-guide_es.md`. This is now ready to review if necessary @lh1008 @erciccione sorry for the trouble with the other PR :)

### CODE REVIEW

File: es/ch02.md [hunk 1]
```diff
@@ -1,90 +1,99 @@
...
 **(4) Clave de gasto:** Tu clave de gasto privado (necesaria para gastar tus fondos).    
 **(5) Altura de restauración (opcional):** Si conoces la altura de bloque en la que se creó tu monedero, puedes especificarla aquí para no tener que escanear toda la cadena de bloques en busca de tus fondos. Por ejemplo, si tu primera transacción se incluyó en el bloque `1350000`, puedes usar una altura ligeramente más baja (por ejemplo,` 1330000`) para que la monedero comience a escanear desde allí, ahorrando tiempo.    
-**(6) Ubicación de la cartera:** Seleccione la carpeta de destino de la cartera.    
+**(6) Ubicación del monedero:** Seleccione la carpeta de destino de la cartera.    
 
 Cuando todo esté listo, haz clic en la flecha derecha y luego en el botón `Usar Monero`.
-En getmonero.org se pone a disposición de los usuarios una guía más avanzada: [[EN] Restoring wallet from keys](https://getmonero.org/resources/user-guides/restore_from_keys.html).
+En getmonero.org se pone a disposición de los usuarios una guía más avanzada: [Restaurar monedero desde claves](https://getmonero.org/resources/user-guides/restore_from_keys.html).
```
Comment (erciccione) [inline es/ch02.md]: If you remove the `[EN]` tag, here. I suggest to link to the spanish version of the userguide on the website

File: es/ch08.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+## Nodos Bootstrap
+
+Un nodo bootstrap es un nodo remoto para usar mientras también se sincroniza la cadena de bloques. Esto difiere a un nodo remoto en que usar un nodo remoto no sincronizará la cadena de bloques a un nodo local.    
+Si se utiliza la característica de nodo bootstrap automáticamente se cambiará al nodo local cuando la cadena de bloques se sincronice totalmente, independientemente de si el usuario eligió el nodo remoto por él mismo o el nodo remoto se especificó por defecto.    
+Más información en https://getmonero.org/resources/moneropedia/bootstrap-node.html.
```
Comment (erciccione) [inline es/ch08.md]: You could link to the spanish moneropedia here

File: es/ch09.md [hunk 3]
```diff
@@ -10,5 +10,6 @@
 + **[[EN] ¿Cómo puedo cambiar el idioma de las 25 palabras de la semilla mnemónica en la interfaz gráfica y en la terminal?](https://monero.stackexchange.com/questions/7373/how-do-i-change-the-language-of-the-25-word-mnemonic-seed-in-the-gui/)**
 + **[[EN] Mi cadena de bloques está bloqueada, ¿cómo la desbloqueo?](https://monero.stackexchange.com/questions/4462/my-blockchain-is-stuck-how-do-i-unstuck-it)**
 + **[[EN] Estoy usando un nodo remoto pero, ¿por qué sigue sincronizándose la cadena de bloques en la interfaz gráfica?](https://monero.stackexchange.com/questions/6324/using-remote-node-still-syncs-blockchain)**
++ **[[EN] Monero GUI v0.13.0.3 no inicia en Linux porque “./monero-wallet-gui: error al cargar librerías compartidas: libhidapi-libusb.so.0”](https://monero.stackexchange.com/questions/10361/gui-monerod-v0-13-0-3-does-not-start-on-linux-because-of-monero-wallet-gui/)**
```
Comment (erciccione) [inline es/ch09.md]: This section is not needed, you can remove it. (see #60 )

File: es/ch02.md (hunk 1 above)
Comment (BlackLotus64) [reply es/ch02.md]: Yes, thanks, I didn't know the link was different :)

File: es/ch08.md (hunk 2 above)
Comment (BlackLotus64) [reply es/ch08.md]: Got it (Y)

**Comment by erciccione** [comment]
Merging this, further changes/reviews can be done opening a new pull request

**Comment by erciccione** [comment]
Thank you for the fixes @BlackLotus64

**Pull request merged**